.github/workflows/tests.yml

```diff
@@ -0,0 +1,99 @@
+name: Tests
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+
+      - name: Set up python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install Poetry
+        uses: snok/install-poetry@v1.1.4
+        with:
+          version: 1.1.6
+          virtualenvs-create: true
+          virtualenvs-in-project: true
+
+      - name: Load cached venv
+        id: cached-poetry-dependencies
+        uses: actions/cache@v2
+        with:
+          path: .venv
+          key: venv-${{ runner.os }}-${{ hashFiles('poetry.lock') }}
+
+      - name: Install dependencies
+        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
+        run: poetry install --no-interaction --no-root
+
+      - name: Run unit tests
+        run: |
+          source .venv/bin/activate
+          PYTHONPATH=src/ poetry run pytest --cov=. --cov-report=xml tests/unit_tests
```

Comment:
Is `PYTHONPATH=src/` doing something?

Reply:
If that's not added it can't find the `mipengine` module. On our computers this is not happening because I guess we have all added the `export PYTHONPATH=...` to our profile.

Reply:
After looking into it again, I removed it and it worked.

Don't remember when it was added and what changed since then, so I can't really answer the question.